source/transactions/tests/README.rst

```diff
@@ -96,34 +105,31 @@ For each YAML file, for each element in ``tests``:
-     "abortTransaction", call the named method on ``session0`` or
-     ``session1``, depending on the "session" argument. 
-   - If ``name`` is "runCommand", call the runCommand method on the database
-     specified in the test. Pass the argument named "command" to the runCommand 
-     method. Pass ``session0`` or ``session1`` to the runCommand method, depending 
-     on which session's name is in the arguments list. If ``arguments`` 
-     contains no "session", pass no explicit session to the method. If ``arguments`` 
-     includes "readPreference", also pass the read preference to the runCommand 
-     method.    
-   - Otherwise, ``name`` refers to a CRUD method, such as ``insertOne``.
-     Execute the named method on the "transactions-tests" database on the "test"
-     collection, passing the arguments listed. Pass ``session0`` or ``session1``
-     to the method, depending on which session's name is in the arguments list.
+   - Create a Database object from the MongoClient, using the ``database_name``
+     field at the top level of the test file.
+   - Create a Collection object from the Database, using the
+     ``collection_name`` field at the top level of the test file.
+     If ``collectionOptions`` is present create the Collection object with the
+     provided options. Otherwise create the object with the default options.
+   - Execute the named method on the provided ``object``, passing the
+     arguments listed. Pass ``session0`` or ``session1`` to the method,
+     depending on which session's name is in the arguments list.
      If ``arguments`` contains no "session", pass no explicit session to the
-     method. If ``arguments`` includes "readPreference", configure the specified
-     read preference in whatever manner the driver supports.
+     method.
    - If the driver throws an exception / returns an error while executing this
      series of operations, store the error message and server error code.
    - If the result document has an "errorContains" field, verify that the
      method threw an exception or returned an error, and that the value of the
-     "errorContains" field matches the error string. If the result document has
-     an "errorCodeName" field, verify that the method threw a command failed
-     exception or returned an error, and that the value of the "errorCodeName"
-     field matches the "codeName" in the server error response.
+     "errorContains" field matches the error string. "errorContains" is a
+     substring (case-insensitive) of the actual error message.
+     If the result document has an "errorCodeName" field, verify that the
+     method threw a command failed exception or returned an error, and that
+     the value of the "errorCodeName" field matches the "codeName" in the
+     server error response.
+     If the operation returns a raw command response, eg from ``runCommand``,
+     then compare only the fields present in the expected result document.
```

Comment:
This is necessary since commands return many fields that we do not care about, opTime, $clusterTime, etc...

Reply:
Acknowledged.  Thanks for adding to the README